packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -113,7 +113,7 @@ void main() {
 
     await tester.tap(find.byType(Icon));
     await tester.pump();
-    await tester.pump(Duration(milliseconds: 10));
+    await tester.pump(const Duration(milliseconds: 10));
```

Comment:
Is it the case that because the lint fired here, that these are not in auto-const scopes?

Reply:
Yes, these expressions are not in a constant context.